src/test/java/com/puppycrawl/tools/checkstyle/api/FullIdentTest.java

```diff
@@ -37,7 +37,16 @@ public void testToString() {
     }
 
     @Test
-    public void testNonValidCoordinates() {
+    public void testNonValidCoordinatesWithNegative() {
+        testNonValidCoordinates(14, 15, "MyTest.MyTestik[15x14]");
+    }
+
+    @Test
+    public void testNonValidCoordinatesWithZero() {
+        testNonValidCoordinates(0, 0, "MyTest.MyTestik[15x14]");
+    }
+
+    private static void testNonValidCoordinates(int columnNo, int lineNo, String expected) {
```

Comment:
Non-test methods shouldn't start with `test`. Rename `test` to `validate`.
Also romani doesn't like assertions outside of test method.